main.tf

```diff
@@ -5,12 +5,13 @@
 ##############################################################################
 
 locals {
-  at_enabled                = var.activity_tracker_crn == null ? [] : [1]
-  metrics_enabled           = var.sysdig_crn == null ? [] : [1]
-  archive_enabled           = var.archive_days == null ? [] : [1]
-  expire_enabled            = var.expire_days == null ? [] : [1]
-  retention_enabled         = var.retention_enabled ? [1] : []
-  object_versioning_enabled = var.object_versioning_enabled ? [1] : []
```

Comment:
From the validation below, it looks like `object_versioning_enabled` removal is intentional right?

Reply:
Yes. Removing the dynamic blocks on object versioning, required by object locking; meant that this local was not use used. Now we always use the variable directly rather than conditionally and do not require the local.